doc/planning_scene_python/src/planning_scene_tutorial.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python
+import sys
```

Comment:
In order to get this to run on my system, I had to change the shbang to `python3`, and also import `rospy`, `moveit_commander`, `rospkg`, and `pathlib`.